### ISSUE TITLE
After fresh install, when I run `app/console sylius:install` on command line, I get stuck in a loop of "This value should not be blank" at stage 3

### ISSUE THREAD

**Issue by blowski** [Sylius/Sylius#2771]
To reproduce:
1. Fresh install from Composer on a Vagrant box with Ubuntu 14.04, default version of PHP5.
2. Run `app/console sylius:install` 
```
Installing Sylius...

Step 1 of 4. Checking system requirements.
+-------------------------------+---------------------------------------------------------------------+
| Issue                         | Recommendation                                                      |
+-------------------------------+---------------------------------------------------------------------+
| short_open_tag                |                                                                     |
| session.auto_start            |                                                                     |
| sylius.extensions.accelerator | Install and enable a PHP accelerator like APC (highly recommended). |
+-------------------------------+---------------------------------------------------------------------+
Success! Your system can run Sylius properly.

Step 2 of 4. Setting up the database.
Creating Sylius database for environment dev.
 4/4 [||||||||||||||||||||||||||||] 100%
Warning! This will erase your database. Your current environment is dev.

Step 3 of 4. Shop configuration.
Please enter list of locale codes, separated by commas or just hit ENTER to use "en_US". For example "en_US, de_DE".
Adding English (United States).
Please enter list of currency codes, separated by commas or just hit ENTER to use "USD". For example "USD, EUR, GBP".
Adding US Dollar.
Please enter list of country codes, separated by commas or just hit ENTER to use "US". For example "US, PL, DE".
Adding United States.
Create your administrator account.
This value should not be blank.
This value should not be blank.
This value should not be blank.
```
That last `This value should not be blank` message repeats over and over again - after about 20 seconds, I did Ctrl+C - I'm assuming it would have gone on forever otherwise.

When I run `app/console sylius:install:setup` it lets me enter locale, currency, and country to sell in, and it works fine. I can then run step 4 manually.

**Comment by patrick-mcdougle** [comment]
:+1: I can reproduce this bug with the current state of the sylius-standard install. Not sure if it's sylius's fault or the fact that the dialog helper is deprecated in symfony 2.6 (which is what composer installs for the standard-edition).

**Comment by bwmarrin** [comment]
I'm also have the exact same problem.  I just tried to install Sylius Standard and got the same loop until PHP ran out of memory.  I am unclear on all of the initial options and if maybe I set one of them wrong because I don't see where there's any documentation about them - so maybe I did something wrong :)

**Comment by songecko** [comment]
same here and also i have another error when sylius:install execute cache:clear command: "Cannot rename "../app/cache/de_" to "../app/cache/dev"

**Comment by rekursed** [comment]
Facing Same problem with OSX 10.10.3

**Comment by nakashu** [comment]
possible fix: [#3071#issuecomment-141682129](https://github.com/Sylius/Sylius/issues/3071#issuecomment-141682129)

**Comment by michalmarcinkowski** [comment]
Does the issue still occur on the current master?
